README.md

```diff
@@ -33,6 +33,17 @@ To deploy with earlier versions of Juju, you will need to provide the cloud
 credentials via the `credentials`, or `access-key` and `secret-key`, charm
 config options.
 
+# Resource Usage Note
+
+By relating to this charm, other charms can directly allocate resources, such
+as EBS volumes and ELBs, which could lead to cloud charges and count against
+quotas.  Because these resources are not managed by Juju, they will not be
+automatically deleted when the models or applications are destroyed, nor will
+they show up in Juju's status or GUI.  It is therefore up to the operator to
```

Comment:
I would be more explicit than "track and audit." What does that mean? "It is therefore up to the operator to manually delete these resources when they are no longer needed, using the AWS console or API."